File: Resource/51900286-51900712-51900718-51900699/source demo/Window Form/51900286-51900712-51900718-51900699/RestaurentDTO/BanAn.cs
using System.Data;

namespace RestaurentDTO
{
    public class BanAn
    {
        private string _maBanAn;
        private string _tinhTrang;
        private int _sucChua;
        private string _ghiChu;

        public BanAn()
        {
            MaBanAn = "";
            TinhTrang = "";
            SucChua = 100;
            GhiChu = "";
        }

        public BanAn(DataRow dataRow)
        {
            this._maBanAn = dataRow["MaBanAn"].ToString();
            this._tinhTrang = dataRow["TinhTrang"].ToString();
            this._sucChua = (int) dataRow["SucChua"];
            this._ghiChu = dataRow["GhiChu"].ToString();
        }

        public BanAn(string maBanAn, string tinhTrang, int sucChua, string ghiChu)
        {
            this._maBanAn = maBanAn;
            this._tinhTrang = tinhTrang;
            this._sucChua = sucChua;
            this._ghiChu = ghiChu;
        }


        public string MaBanAn { get => _maBanAn; set => _maBanAn = value; }
        public string TinhTrang { get => _tinhTrang; set => _tinhTrang = value; }
        public int SucChua { get => _sucChua; set => _sucChua = value; }
        public string GhiChu { get => _ghiChu; set => _ghiChu = value; }
    }
}

File: Dev/Models/Models/BanAn.java
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
 
@Entity
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
@Table(name = "banan")
public  class BanAn {
    @Id
    @GeneratedValue
    private Long MaBanAn;

    private String TinhTrang;
    
    private int SucChua;
    
    private String GhiChu;
}